services/spanishUtils.js

```diff
@@ -0,0 +1,186 @@
+/*
+Utilities to supplement conjugation library.
+*/
+
+
+/*shows what a verb would look like if it wasn't irregular. wraps the conjugation logic from the conjuation library 
+but decouples from irregularity.
+*/
+
+const conjugate = require('./conjugation');
+
+var verb_endings = [
+    "ar", "er", "ir",
+    "arse", "erse", "irse",
+    "ár", "ér", "ír",
+    "árse", "érse", "írse"
+]
+
+var ir_endings = {
```

Comment:
This was a bigger question I had. Is there a way to use the internal variables of conjugation.js? Initially, I was trying to avoid changing the library. What do you think? 

I had to make pull out some functionality to be able to do predicted conjugations (assuming the verb wasn't irregular).